app/scripts/controllers/dashboard.coffee

```diff
@@ -158,6 +158,7 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', [
 
         $scope.setSchool = (item, model, showAllSchools) ->
             $scope.selectedSchool = item
+            $scope.selectedSchool['pupil-teacher-ration'] = 56
```

Comment:
pt_ratio was always null. I hardcoded to test it. we need to change the data anyway